arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -338,7 +331,7 @@
         "tags": [
           "WorkspaceCollections"
         ],
-        "operationId": "WorkspaceCollections_listBySubscription",
+        "operationId": "WorkspaceCollections_list",
```

Comment:
sure, if you update the current one it would be. Regarding autorest, we've been updating rules, just be sure to update to the latest https://github.com/Azure/autorest#updating-autorest , which should match what's reported by CI.